### PR TITLE
tests(entities): add search that includes a synthetics entity

### PR DESCRIPTION
**Pull request by zlesnr**

### CODE REVIEW

**Comment by zlesnr** [comment]
Relates to https://github.com/newrelic/newrelic-client-go/issues/612

**Review by ctrombley — APPROVED** [review]

File: .tutone.yml [hunk 1]
```diff
@@ -592,6 +592,8 @@ packages:
         create_as: string
       - name: Seconds
         create_as: string
```
Comment (jthurman42) [inline .tutone.yml:594]: Do we know that `Seconds` as string won't bite us either?

Comment (zlesnr) [reply .tutone.yml:594]: We do not.  Minutes is a scalar, but comes back in the JSON as an int.

Comment (zlesnr) [reply .tutone.yml:594]: Seconds also is a scalar.  I'll look for some usage of it.

Comment (zlesnr) [reply .tutone.yml:594]: https://github.com/newrelic/newrelic-client-go/issues/617

**Pull request merged**